implementation.js

```diff
@@ -14,29 +14,41 @@ var $toString = callBound('Object.prototype.toString');
 var stackDesc = gOPD($Error.prototype, 'stack');
 var stackGetter = stackDesc && stackDesc.get && callBind(stackDesc.get);
 
+var domExceptionClonable = !!($structuredClone && $structuredClone(new DOMException()) instanceof $Error);
+
 module.exports = function isError(arg) {
 	if (!arg || (typeof arg !== 'object' && typeof arg !== 'function')) {
```

Comment:
Out of interest, why is `function` also allowed? Is this ES3 compat or something?

Reply:
Just mirroring the spec text; a function can be an object too.

It's a good point that there aren't any implementations I'm aware of where Error objects can be functions.